rule-packs/azure.json

```
@@ -0,0 +1,388 @@
+[
+    {
+      "name": "azure-asc-monitor-system-updates",
+      "description": "Finds if Azure Security Center setting 'Monitor System Updates' is disabled.",
```

Forgive me if I sound like a broken record or am otherwise missing some context...

What do you think about making these similar to the `gcp.json` descriptions, which take the form "Finds {{ entities such as 'Compute instances' }} {{ meeting some criteria }}"? This would be in contrast to the the form "Finds if ... is {{ something }}", which to me seems to indicate it is going to return a boolean value.

In this specific case, I would recommend something like:
```suggestion
      "description": "Finds Azure Security Center policies with the 'Monitor System Updates' setting disabled.",
```